extensions/content-health-monitor/manifest.json

```
@@ -5,10 +5,10 @@
     "appmode": "quarto-static"
   },
     "extension": {
-    "name": "app-canary",
-    "title": "App Canary",
-    "description": "Provides a way to monitor deployed content and optionally email a report when any of the content has failed.",
-    "homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/app-canary",
+    "name": "content-health-monitor",
+    "title": "Content Health Monitor",
+    "description": "This report uses the publisher’s API key to monitor a single piece of content. It checks whether the content is reachable, but does not validate its functionality. If scheduled to run regularly, it will send an email alert if the content becomes unreachable.",
```

If you take the "if" suggestion above you can take this one too

```suggestion
    "description": "This report uses the publisher’s API key to monitor a single piece of content. It checks whether the content is reachable, but does not validate its functionality. When scheduled to run regularly, it will send an email alert if the content becomes unreachable.",
```